styles/rrt.go

```diff
@@ -6,13 +6,27 @@ import (
 
 // Rrt style.
 var Rrt = Register(chroma.MustNewStyle("rrt", chroma.StyleEntries{
-	chroma.Comment:        "#00ff00",
-	chroma.NameFunction:   "#ffff00",
-	chroma.NameVariable:   "#eedd82",
-	chroma.NameConstant:   "#7fffd4",
-	chroma.Keyword:        "#ff0000",
-	chroma.CommentPreproc: "#e5e5e5",
-	chroma.LiteralString:  "#87ceeb",
-	chroma.KeywordType:    "#ee82ee",
-	chroma.Background:     " bg:#000000",
```

Comment:
You'll need to add a foreground colour to `chroma.Background`. It's a bit confusing that it's called "Background" to be honest, as it's actually the "default" style if nothing else exists. Without it, any tokens without a style will fall back to `Background` and render without a foreground colour (confusing).

Reply:
Thanks for the feedback, I didn't know that. I've fixed this in the latest commit.